packages/web/next.config.js

```diff
@@ -13,8 +13,15 @@ module.exports = withTM(
         },
       ];
     },
-    future: {
```

Comment:
I'm not sure this should be removed.
What's the reason for removing this and isn't this what dysbulic added to use webpack 5 for Glaze?

Reply:
did it by mistake while resolving merge conflicts

Reply:
there is a build error now after adding this 
```
Module not found: Error: Can't resolve 'fs' in 
```